lib/bolt/config/src/ns.rs

```diff
@@ -663,3 +665,11 @@ fn default_minio_port() -> u16 {
 fn default_tunnel_port() -> u16 {
 	5000
 }
+
+#[derive(Serialize, Deserialize, Clone, Debug, Default)]
+#[serde(deny_unknown_fields)]
+pub struct BetterUptime {
+	pub company_name: String,
+	pub company_url: String,
+	pub company_subdomain: String,
```

Comment:
Can we add some brief inline docs on this? We don't have external docs, but it'd be nice to have inline docs at bare minimum. For example, `company_subdomain` is not immediately obvious what it does. Might help to include a quick example, up to you.